common/reviews/api/rush-sdk.api.md

```diff
@@ -0,0 +1,39 @@
+## API Report File for "@rushstack/rush-sdk"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+/// <reference types="node" />
+
+// @public
+export interface ILoadSdkAsyncOptions {
+    abortSignal?: AbortSignal;
+    onNotifyEvent?: SdkNotifyEventCallback;
+    rushJsonSearchFolder?: string;
+}
+
+// @public
+export interface IProgressBarCallbackLogMessage {
+    kind: 'info' | 'debug';
+    text: string;
+}
+
+// @public
+export interface ISdkCallbackEvent {
+    logMessage: IProgressBarCallbackLogMessage | undefined;
+    progressBarPercent: number | undefined;
+}
+
+// @public
+export class RushSdkLoader {
+    static get alreadyLoaded(): boolean;
```

Comment:
```suggestion
    static get isLoaded(): boolean;
```

Reply:
I thought `isLoaded()` is used for functions whereas `loaded` is used for properties (?)

Reply:
After thinking about it some more, I renamed it